pkgs/development/tools/convox/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, fetchurl, buildGoPackage }:
+
+buildGoPackage rec {
+  name = "convox-${version}";
+  version = "20171002150509";
+
+  meta = {
+    homepage = https://convox.com;
+    description = "Open-source PaaS";
+    license = lib.licenses.asl20;
+    platforms = lib.platforms.linux ++ lib.platforms.darwin;
+  };
```

Comment:
A maintainer must be set.

Reply:
What’s the right way to set that? Should I add myself as a maintainer?

Reply:
Yep, just add yourself to [lib/maintainers.nix](https://github.com/NixOS/nixpkgs/blob/master/lib/maintainers.nix), and then add a `maintainers = with lib.maintainers; [ shosti ];` entry to `meta`.